examples/Reranking_search_results_with_cross-encoders.ipynb

```diff
@@ -0,0 +1,805 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7f30b8b2",
+   "metadata": {},
+   "source": [
+    "# Reranking search results with cross-encoders\n",
+    "\n",
+    "This notebook takes you through examples of using a cross-encoder to re-rank search results.\n",
+    "\n",
+    "This is a common use case with our customers, where you've implemented semantic search using embeddings (produced using a [Bi-encoder](https://www.sbert.net/examples/applications/retrieve_rerank/README.html#retrieval-bi-encoder)) but the results are not exactly what you need. A possible cause is that there is some business rule you can use to rerank the documents such as how recent or how popular a document is. \n",
+    "\n",
+    "However, often there are subtle domain-specific rules that help determine relevancy, and this is where a cross-encoder can be useful. Cross-encoders are more accurate than bi-encoders but they don't scale well, so using them to re-order a shortened list returned by semantic search is the ideal use case.\n",
+    "\n",
+    "We'll use text-davinci-003 with logprobs to build a cross-encoder here.\n",
```

Comment:
optional: could be worth emphasizing that GPT's advantage of being a cross-encoder is its language understanding

Reply:
Good point, have added a sentence after the mention of `text-davinci-003`